packages/web-forms/src/components/widgets/IconSVG.vue

```diff
@@ -0,0 +1,125 @@
+<script setup lang="ts">
```

Comment:
Lint enforces 2 word names. I considered many names, I wanted to avoid `UI`, `App`, `Widget`, `Component`, `Base`, this one seems descriptive and short. 
I take suggestions

Reply:
`MaterialIcon`? `MaterialDesignIcon`? `MdIcon`?

Or is the idea that there could possibly multiple icon sources? If so, `IconSVG` feels fine.

Reply:
Thanks! It may be unlikely to have different sources, but `IconSVG` seems open enough, I will leave this name for now.